README.md

```diff
@@ -606,6 +606,17 @@ Some of the external audit tools use this standard. For example Nessus has funct
     ```bash
     echo "net.ipv4.icmp_echo_ignore_broadcasts = 1" > /etc/sysctl.d/50-net-stack.conf
     ```
+## Disable SACK
+- <img src="https://github.com/trimstray/linux-hardening-checklist/blob/master/static/img/medium.png" alt="medium"> Disable TCP SACK.
+
+    **Example:**
+    
+    ```bash
+    sysctl -w net.ipv4.tcp_sack=0
+    echo "net.ipv4.tcp_sack = 0" | sudo tee /etc/sysctl.conf
```

Comment:
this is a bad idea for a line people might just copy and paste. It would replace sysctl.conf.

Reply:
Yep, you're right.  Changed to `tee -a`.